simapp/app.go

```diff
@@ -133,6 +134,36 @@ var (
 	_ servertypes.Application = (*SimApp)(nil)
 )
 
+// stdAccAddressCodec is a temporary address codec that we will use until we
```

Comment:
does this have to be in app.go or can it be somewhere users can import to get the needed information?

Reply:
We can just refactor this to use `address.Bech32Codec`. Does how I've updated it in the latest commit look okay for now? It's still using the global but we can refactor that later.